docs/src/doc/docs/user_guide/gradle/usage.md

```diff
@@ -0,0 +1,293 @@
+            // Repeat for multiple sourceRoots
+            sourceRoot.from(file("src"))
+
+            // Specifies the location of the project source code on the Web.
+            // If provided, Dokka generates "source" links for each declaration.
+            // Repeat for multiple mappings
+            sourceLink {
+                // Unix based directory relative path to the root of the project (where you execute gradle respectively). 
+                localDirectory.set(file("src/main/kotlin"))
+
+                // URL showing where the source code can be accessed through the web browser
+                remoteUrl.set(java.net.URL(
+                    "https://github.com/cy6erGn0m/vertx3-lang-kotlin/blob/master/src/main/kotlin"))
+                // Suffix which is used to append the line number to the URL. Use #L for GitHub
+                remoteLineSuffix.set("#L")
+            }
+
+            // Used for linking to JDK documentation
+            jdkVersion.set(8)
+
+            // Disable linking to online kotlin-stdlib documentation
+            noStdlibLink.set(false)
+
+            // Disable linking to online JDK documentation
+            noJdkLink.set(false)
+
+            // Disable linking to online Android documentation (only applicable for Android projects)
+            noAndroidSdkLink.set(false)
+
+            // Allows linking to documentation of the project"s dependencies (generated with Javadoc or Dokka)
+            // Repeat for multiple links
+            externalDocumentationLink {
+                // Root URL of the generated documentation to link with. The trailing slash is required!
+                url = URL("https://example.com/docs/")
+
+                // If package-list file is located in non-standard location
+                // packageListUrl = URL("file:///home/user/localdocs/package-list")
+            }
+
+            // Allows to customize documentation generation options on a per-package basis
+            // Repeat for multiple packageOptions
+            perPackageOption {
+                prefix.set("kotlin") // will match kotlin and all sub-packages of it
+                // All options are optional, default values are below:
+                skipDeprecated.set(false)
+                reportUndocumented.set(true) // Emit warnings about not documented members 
+                includeNonPublic.set(false)
+            }
+            // Suppress a package
+            perPackageOption {
+                prefix.set("kotlin.internal") // will match kotlin.internal and all sub-packages of it
+                suppress.set(true)
+            }
+        }
+    }
+```
+
+## Multiplatform
+Dokka supports single-platform and multi-platform projects using source sets abstraction. For most mutli-platform projects
+you should assume that dokka's source sets correspond to Kotlin plugin's source sets. Main source sets will be registered
```

Comment:
All source sets are now automatically registered. Test source sets will just be suppressed!

Reply:
Done ✅